packages/app/src/components/Admin/Customize/CustomizeLogoSetting.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { FC, useState } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+
+import AdminCustomizeContainer from '~/client/services/AdminCustomizeContainer';
+import AppContainer from '~/client/services/AppContainer';
+import {
+  toastError, toastSuccess,
+} from '~/client/util/apiNotification';
+import ImageCropModal from '~/components/Common/ImageCropModal';
+
+import { withUnstatedContainers } from '../../UnstatedUtils';
+import AdminUpdateButtonRow from '../Common/AdminUpdateButtonRow';
+
+
+type Props = {
+  adminCustomizeContainer : AdminCustomizeContainer
+}
+
+const CustomizeLogoSetting: FC<Props> = (props: Props) => {
+
+  const { t } = useTranslation();
+  const { adminCustomizeContainer } = props;
+  const [isShow, setIsShow] = useState<boolean>(false);
+  const [src, setSrc] = useState<ArrayBuffer | string | null>(null);
+  const {
+    uploadedLogoSrc, isUploadedLogo, isDefaultLogo, defaultLogoSrc,
+  } = adminCustomizeContainer.state;
+
+  const hideModal = () => {
+    setIsShow(false);
+  };
+
+  const cancelModal = () => {
+    hideModal();
+  };
+
+  const onSelectFile = (e) => {
+    if (e.target.files != null && e.target.files.length > 0) {
+      const reader = new FileReader();
+      reader.addEventListener('load', () => setSrc(reader.result));
+      reader.readAsDataURL(e.target.files[0]);
+      setIsShow(true);
+    }
+  };
+
+  const onClickSubmit = async() => {
```

Comment:
If you update twice in a row, the radio button with the default logo will be selected.

This needs fixing